src/layout/Footer.tsx

```diff
@@ -1,28 +1,50 @@
-import React from 'react';
+import React, { FC } from 'react';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
 import logo from '../resources/images/unit_logo.png';
+import { Link } from 'react-router-dom';
+import { Link as MuiLink } from '@material-ui/core';
+import NormalText from '../components/NormalText';
 
 const StyledFooter = styled.div`
-  display: flex;
+  display: grid;
+  grid-template-areas: '. logo privacy';
+  grid-template-columns: 1fr 1fr 1fr;
   min-height: 3rem;
-  font-size: 1rem;
-  font-weight: bold;
-  justify-content: center;
   align-items: center;
   border-top: 2px solid ${({ theme }) => theme.palette.separator.main};
 `;
 
-const StyledLogo = styled.img`
-  margin-left: 0.3rem;
+const StyledLogoContainer = styled.div`
+  grid-area: logo;
+  justify-self: center;
 `;
 
-const Footer: React.FC = () => {
+const StyledPrivacyPolicyContainer = styled.div`
+  grid-area: privacy;
+  justify-self: end;
+  padding-right: 0.5rem;
+`;
+
+const Footer: FC = () => {
   const { t } = useTranslation('');
+
   return (
     <StyledFooter>
-      {t('delivered_by')}
-      <StyledLogo src={logo} alt="UNIT logo" />
+      <StyledLogoContainer>
+        <NormalText>{t('delivered_by')}</NormalText>
```

Comment:
ingen namespace her eller line 30?

Reply:
det er gammelt, så det ligger i `translations.json`. Skal jeg flytte de til common?

Reply:
kan du godt. translations.json kan gravlegges for min del